pkg/util/pki/parse_test.go

```diff
@@ -288,6 +288,11 @@ func TestParseSingleCertificateChain(t *testing.T) {
 			expPEMBundle: PEMBundle{ChainPEM: joinPEM(leaf.pem, intA2.pem, intA1.pem), CAPEM: root.pem},
 			expErr:       false,
 		},
+		"if only CAs are passed, should still output the best chain possible": {
+			inputBundle:  joinPEM(root.pem, intA2.pem, intA1.pem),
+			expPEMBundle: PEMBundle{ChainPEM: joinPEM(intA2.pem, intA1.pem), CAPEM: root.pem},
+			expErr:       false,
+		},
```

Comment:
self-review note: this was added for defence in depth because I asked myself the question if this case would work correctly when I was doing the "experimental" commit. I had no reason to suspect this wouldn't work, but it seemed a worthwhile test to add.